DataFormats/METReco/src/MET.cc

```diff
@@ -68,6 +68,24 @@ MET::MET( double sumet_, const std::vector<CorrMETData>& corr_,
     }
 }
 
+// Copy constructor
+//____________________________________________________________________________||
+MET::MET( const MET& met ): 
```

Comment:
is this overload really doing anything other than the default copy constructor?
~OK to stay